mappings/com/mojang/blaze3d/shader/GlShader.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/unmapped/C_chbhvqie com/mojang/blaze3d/shader/GlShader
+	METHOD m_byoiplce getId ()I
+	METHOD m_lzgywito markUniformsDirty ()V
+	METHOD m_nfebcxtw attachReferencedShaders ()V
+	METHOD m_qnzasjah getFragmentProgram ()Lnet/minecraft/unmapped/C_bejltllu;
```

Comment:
First time hearing Program for a specific step of a shader program. Shouldn't it be `getFragmentShader`?

From the standard vocabulary of graphics stuff, it feels like this class is misnamed and is supposed to be named `GlShaderProgram` since shader alone is usually *a step* of the shader program.

Reply:
This is the only thing left to resolve. Let me ask some community members to input on this

Reply:
> I agree with Aurora, getFragmentShader, not sure about the class change though. GlShaderProgram or GlShader would work for me.

These are two different things. The program holds all of the shaders, which are the individual programmable stages. The method `getFragmentProgram` should be `getFragmentShader` and the class should either be `GlShaderProgram` or `GlProgram`